src/Clp.jl

```diff
@@ -31,4 +31,12 @@ end
 
 include("moi/MOI_wrapper.jl")
 
+# TODO(odow): remove at Clp.jl v1.0.0.
+function ClpSolver(args...; kwargs...)
+    error(
+        "`ClpSolver` is no longer supported. Upgrade to JuMP 0.20 or greater," *
```

Comment:
JuMP 0.19 or greater? Also not everyone calling ClpSolver() is necessarily using JuMP. They could be using `linprog` or another package that uses MPB.

Reply:
How about ````"`ClpSolver` is not longer supported. If you are using JuMP, upgrade to the latest version and use `Clp.Optimizer` instead. If you are using MathProgBase (e.g., via `lingprog`), you will need to upgrade to MathOptInterface (https://github.com/JuliaOpt/MathOptInterface.jl)."````

I'm guessing the number of users that are going to hit this is very small.

Reply:
Sounds good. The number is small but almost definitely nonzero.